test/portal_test.dart

```diff
@@ -59,7 +60,7 @@ void main() {
           isPrivate: false,
         ),
       ]));
-  when(mockWebsiteProvider.fetchFavouriteWebsites()).thenAnswer(
+  when(mockWebsiteProvider.fetchFavouriteWebsites(mockAuthProvider.uid)).thenAnswer(
```

Comment:
You should read more about how mocking works here: https://pub.dev/packages/mockito

For this use case, the correct syntax is `fetchFavouriteWebsites(any, uid: anyNamed('uid'))`.